templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <div class="btn-group">
+                  <a class="btn btn-sm btn-outline-secondary" href="{% url "taxi:manufacturer-update"ff manufacturer.id %}">Update</a>
+                  <a class="btn btn-sm btn-outline-danger" href="{% url "taxi:manufacturer-delete" manufacturer.id %}">Delete</a>
```

Comment:
Consider using single quotes in the `{% url %}` template tag: `{% url 'taxi:manufacturer-delete' manufacturer.id %}`. This is a recommended practice to avoid conflicts with HTML attributes that use double quotes.